src/replit_river/codegen/run.py

```diff
@@ -38,9 +39,27 @@ def main() -> None:
         action="store_true",
         default=False,
     )
+    client.add_argument(
+        "--protocol-version",
+        help="Generate river v2 clients",
+        action="store",
+        default="v1.1",
+        choices=["v1.1", "v2.0"],
+    )
+    client.add_argument(
+        "--method-filter",
+        help="Only generate a subset of the specified methods",
```

Comment:
help string should probably include that this is a path (unless `type=pathlib.Path` already does that)

Reply:
in fact it does